middleware/jwt/src/state_data.rs

```diff
@@ -0,0 +1,21 @@
+pub use jsonwebtoken::TokenData;
+
+/// Struct to contain the JSON Web Token on a per-request basis.
+#[derive(StateData)]
+pub struct AuthorizationToken<T>
+where
+    T: Send + 'static,
+{
+    /// Headers, Claims, and other data from the token.
+    pub token: TokenData<T>,
```

Comment:
I don't think this field needs to be public, we can direct people to `new` instead.

Reply:
This field has been updated in the new version, pushed just before this comment.